Poker.BE/Poker.BE.Domain/Game/GameConfig.cs

```diff
@@ -1,217 +0,0 @@
-﻿using Poker.BE.Domain.Game;
-using System;
-using System.Collections.Generic;
-
-namespace Poker.BE.Domain.Game
-{
-    public class GameConfig
```

Comment:
I don't believe so... only one thing: I changed MaxNumOfPlayers to be the max number of active players, and removed MaxNumOfActivePlayers.
the concept is exactly the same, the DB still saves the same parameters...
I changed only the code design - GameConfig became NoLimitHoldem and now inherits GamePreferences.
All of the fields ramains the same.